src/index.html

```diff
@@ -7,7 +7,53 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page">
```

Comment:
you may not add a class to a body and use a tag selector directly since it's only one tag on a page 
```suggestion
  <body>
```